### ISSUE TITLE
/Bug/The menu of network on the task bar does not work

### ISSUE THREAD

**Issue by JiachengLiu13579** [cairoshell/cairoshell#211]
We all know that in explorer.exe, if we click on the button on the taskbar that is either the icon for wireless network or Ethernet, a sub menu would pop up, allowing us to connect/disconnect from wireless not works and see a general status of the connection. But with Cairo in Windows 7, no matter how many times I click on that icon, the sub menu just won’t pop up. And since that’s the only way to connect to a differnt wireless network, I have to quit Cairo every time when I want to change the connection or connect to a new network. That has happened ever since I get Cairo.

**Comment by JiachengLiu13579** [comment]
To elucidate my point, I mean, this icon is unresponsive: 

<img width="346" alt="captura3" src="https://user-images.githubusercontent.com/43326954/46715868-58b0ad80-cc94-11e8-861e-1bfc04341298.PNG">

**Comment by josuave** [comment]
Is that the windows native network icon?

**Comment by JiachengLiu13579** [comment]
Yes. The only way to configure WiFi connection in Windows 7 is through this sub menu (the control panel settings would bring you to the sub menu as well). It’s interesting that if I click on the better icon, it gives me the sub menu of power options (“high performance,” “balance,” etc.). The sub menu for the battery works but the one for network doesn’t.

**Comment by dremin** [comment]
This is something I have struggled to make work.. there's an old issue that was submitted to make these small config applets within Cairo rather than using native system ones, which would solve this but there are some prerequisites that have to be in place before starting this work. Of course, if someone else (or future me) finds a solution to this with the native network applet, it will be explored!

**Comment by josuave** [comment]
I was thinking, rather than building a taskbar functionality/icon, why don’t we attempt using tray icons to achieve these features. There is no need to even tie them to this implementation of the taskbar area. All we would need to do is, on left or right click, show a menu or wpf window. They could be implemented either in Cairo itself or as extensions.

**Comment by josuave** [comment]
Duplicate of #146

**Comment by dremin** [comment]
Closing as duplicate
